tests/PythonTests.cmake

```diff
@@ -103,6 +103,12 @@ function(add_python_test case)
     set(other_covg "")
   endif()
 
+  if(fn_COVERAGE_PATHS)
+    set(coverage_source_paths "${fn_COVERAGE_PATHS}")
```

Comment:
I could go either way.  I did this to make it easy to uninclude default paths.  I think the same could be achieve with an `omit` in the coveragerc file.